README.md

```diff
@@ -1,221 +1,172 @@
-set arg0=%0
-if [%arg0:~2,1%]==[:] if not [%TERM_PROGRAM%] == [vscode] pause
-:end
-```
+To install the tools go to the folder `frank-flow/src/frontend` this is needed because of the way this mono repo has been structured. This might change in the feature. Run `yarn` in this folder and everything should be good to go!
 
-### Frank!Flow
+### Frontend
 
-Clone this repository (or a fork) to start developing the Frank!Flow. This project consists of two main parts: The JavaScript frontend and the Java backend.
+The code for the frontend is located at `frank-flow/src/frontend`. This folder contains the cypress integration tests, the source files for the application and configurations. 
 
-#### Requirements
+#### Building the frontend
 
-There are some basic requirements that are needed to test or develop the Frank!Flow application. These requirements are:
+There are two ways to build the application, for development and for production. The difference lays in the way the code gets transpiled.
 
-- [Java](https://www.java.com/nl/download/)
-- [Node.js](https://nodejs.org/en/)
-- [Maven](http://maven.apache.org/)
+##### Development
 
-#### Frontend
+The dist folder is located at  `frank-flow/target/frontend`, set the absolute path of this folder as [frontend path in the properties file](#configuration) so the Frank!Runner knows what to serve. To build a development version you can run `yarn build:dev` or `yarn watch` if you want to build continually. The code won’t be minified which will make it easier to debug.
 
-The code for the frontend is located at `frank-flow/src/frontend`. The most important files in this folder are the index.html which bootstraps the application and the app folder which contains all the logic of the application. Some other notable folders are: css for the stylesheets, media for images and resources and test for the Karma tests.
+##### Production
 
-##### Developing the frontend
+You can make a production build by running the command `yarn build:prod` or by [building the whole application](#building-the-whole-application).
 
-Start by installing the node modules with the command `npm install`. This will download the required packages that the Frank!Flow depends on. Some of these packages are: jsPlumb for the flow canvas and Monaco-editor for the build in editor.
+##### Analyze build
 
-After installing the node_modules you can start coding! You can automatically build the files while you are editing by using the command `npm run watch`, however you will need to refresh the browser yourself.
+To analyze the size of the build and which files are making it so big, run `yarn build:analyze`.
 
-##### Building the frontend
+### Backend
 
-There are two ways to build the application, for development and for production. The difference lays in the build target and build mode.
+The backend is used to serve the frontend and as an API for getting files from the file-system. The code is located at `frank-flow/main/java`.
 
```

Comment:
I think the code is located in frank-flow/src/main/java

Reply:
You're right

Reply:
Ok.